recipes-extended/xen/files/xen-x86-Fix-up-rules-when-forcing-mno-sse.patch

```diff
@@ -36,12 +36,12 @@ PATCHES
 ################################################################################
 --- a/xen/arch/x86/Rules.mk
 +++ b/xen/arch/x86/Rules.mk
-@@ -25,7 +25,7 @@ $(call as-insn-check,CFLAGS,CC,".equ \"x
-                      '-D__OBJECT_LABEL__=$(subst $(BASEDIR)/,,$(CURDIR))/$$@')
- $(call as-insn-check,CFLAGS,CC,"invpcid (%rax)$$(comma)%rax",-DHAVE_AS_INVPCID)
+@@ -29,7 +29,7 @@ $(call as-option-add,CFLAGS,CC,"invpcid
+ $(call as-option-add,CFLAGS,CC,\
+     ".if ((1 > 0) < 0); .error \"\";.endif",,-DHAVE_AS_NEGATIVE_TRUE)
  
--CFLAGS += -mno-red-zone -mno-sse -fpic
-+CFLAGS += -mno-red-zone -mno-sse -fpic -mfpmath=387
- CFLAGS += -fno-asynchronous-unwind-tables
- # -fvisibility=hidden reduces -fpic cost, if it's available
- ifneq ($(call cc-option,$(CC),-fvisibility=hidden,n),n)
+-CFLAGS += -mno-red-zone -fpic -fno-asynchronous-unwind-tables
++CFLAGS += -mno-red-zone -fpic -fno-asynchronous-unwind-tables -mfpmath=387
+ 
+ # Xen doesn't use SSE interally.  If the compiler supports it, also skip the
+ # SSE setup for variadic function calls.
```

Comment:
Looks like '-mno-sse' moved to just after this patch context.  '-mfpmath=387' may make more sense being added there.
```
CFLAGS += -mno-red-zone -fpic -fno-asynchronous-unwind-tables

# Xen doesn't use SSE interally.  If the compiler supports it, also skip the
# SSE setup for variadic function calls.
CFLAGS += -mno-sse $(call cc-option,$(CC),-mskip-rax-setup)
```

Reply:
New PR up with this fixed: https://github.com/OpenXT/xenclient-oe/pull/1069